### DISCUSSION TITLE
Add support for Zilliqa mining, dualmining and staking

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1171] [Ideas]
Many Miners offers the possibility to dualmine Ethereum and Zilliqa. Beside mining, Zilliqa also uses PoS. Therefore, it be great when Miningcore supports this features too. 

As far as I understood the Zilliqa documentation, the miningpart is based on ethash. Probably, this would be easy to implement :).

**Comment by jon4hz** [comment]
ZIL support would be awesome indeed!

**Reply by oliverw** [reply]
Looking for contributors as I'm totally swamped in work right now.

**Reply by maximummix** [reply]
I am ready to purchase this development of the ETC-ZIL pool at a price https://store.miningcore.pro/product/miningcore-additional-currency/

**Reply by oliverw** [reply]
This is not what you think it is. Read the product description again.
